scripts/resolve-pr-threads.sh

```
@@ -381,6 +465,352 @@ if [ "${RESOLVE_PR_THREADS_SKIP_IDENTITY_CHECK:-0}" != "1" ] && ! $DRY_RUN; then
+# resolve mutation, no separate gate needed.
+post_tag_reply() {
+  local thread_id="$1"
+  local class="$2"
+  local rationale="$3"
+  local body
+  body="[mergepath-resolve: $class] $rationale"
+  # Suppress stdout (the mutation response is noise), but capture
+  # stderr for failure-mode logging. `2>&1 1>/dev/null` swaps
+  # streams so we can grep the diagnostic on failure.
+  local err
+  if ! err=$(gh_pat api graphql \
```

Shellcheck (0.11.0)

[warning] err appears unused. Verify use (or export if used externally).

(SC2034)